config/datocms/scripts/seed-docs.ts

```diff
@@ -0,0 +1,214 @@
+import { readdir, readFile } from 'node:fs/promises';
+import path from 'node:path';
+import dotenv from 'dotenv-safe';
+import { buildClient } from '@datocms/cma-client-node';
+import { fromMarkdown } from 'mdast-util-from-markdown';
+import type { Root } from 'mdast';
+import { toHast } from 'mdast-util-to-hast';
+import { hastToStructuredText, type HastRootNode } from 'datocms-html-to-structured-text';
```

Comment:
Astro check in the CI lint job fails on this module. Not sure why it's not found. Locally I can verify it exists. Maybe it has to do with the `"type": "commonjs"` in `package.json`? But I can't change that to `"module"` as then running DatoCMS migrations will fail 🤷 .

It's a bit of a non-issue. Astro check shouldn't even lint this file and just stick to `src/`. But I'm unable to configure that. What I could do as a workaround is in the CI lint job, delete `config/datocms/scripts/` before running `npm run lint:astro`. What do you think?

Reply:
patch available as PR: https://github.com/voorhoede/head-start/pull/241